docs/compiling.rst

```diff
@@ -1,6 +1,38 @@
+.. _compiling:
+
 Build systems
 #############
 
+Building manually
+=================
+
+pybind11 is a header-only-library, hence it is not necessary to link against
+any special libraries (other than Python itself).
+
+On Linux, the above example can be compiled using the following command:
```

Comment:
There is no "above" anymore.  Perhaps:
```
On Linux, you can compile an example such as the ones given in
:ref:`simple_example` using the following command:
```

with an addition of:

```
.. _simple_example:
```
in `basics.rst` just before:
```
Creating bindings for a simple function
=======================================
```